ifStatement.js

```diff
@@ -47,7 +64,19 @@ Steps:
     - If the number is zero, print a message saying that it is zero.
 ************************************************************************************************/
 // TODO: ADD YOUR CODE BELOW
-
+let number=prompt("enter namber");
+let float=parseInt(number);
+
+if ( float >= 0) 
+{
+  console.log("the number is positive ")
+}
+ if ( float <= 0)
+{
+  console.log("the number is negatine ")
+
+}
+else console.log("is zero ");
```

Comment:
This will not work properly because you set the conditions <= and >=, which means greater than or equal to zero. When you enter a zero number, the first condition will be met, and it will not pass to the else statement.